src/test/java/org/dz/solution/task33/Task33Test.java

```diff
@@ -0,0 +1,66 @@
+package org.dz.solution.task33;
+
+import org.junit.Test;
+
+import java.io.*;
+import java.util.*;
+
+
+public class Task33Test {
+    private static final String SOURCE_DATA_FILE = "src/main/resources/org/dz/solution/task33/MainFile";
+    private static final String RESULT_FILE = "src/main/resources/org/dz/solution/task33/OutputFile";
+    private static final String[] SYMBOLS = new String[]{",", "-", "\\.", ";", "(", ")", ":", "\"", "«", "»", "„", "“"};
+    public static final String WORD_DELIMITER = " ";
+
+    @Test
+    public void test() {
+    }
+
+    private void calculateSkippedNumbers() {
```

Comment:
Numbers??? Why are you using a name from the previous task. Please rename the method. (calculateSkippedNumbers)